Second Pull Request.

harisahmed11 opened the pull request:
Create Event and Drop Down Issue Resolved.

adnanshussain commented:
This Pull request is too complicated. there are too many times the same files have been edited and added and deleted.

Please submit a simpler pull request. Kaam aik file ka tha, changes 10 files main ki hain.

harisahmed11 commented:
Ok sir i will re generate the pull request, and also include task 5 in it
the task is completed.

On Aug 8, 2016 10:36 AM, "Adnan Hussain" <notifications@github.com> wrote:

> This Pull request is too complicated. there are too many times the same
> files have been edited and added and deleted.
>
> Please submit a simpler pull request. Kaam aik file ka tha, changes 10
> files main ki hain.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/adnanshussain/FintechPy/pull/5#issuecomment-238145725>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AS4N6-vpbGVfsXJlHXqVkX5mv0800kMDks5qdsBWgaJpZM4Jd2UW>
> .
>
